jdi-light-material-ui-tests/src/main/java/io/github/com/pages/datadisplay/IconsFrame.java

```diff
@@ -0,0 +1,22 @@
+package io.github.com.pages.datadisplay;
+
+import com.epam.jdi.light.elements.composite.Section;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.UI;
+import com.epam.jdi.light.material.elements.datadisplay.Icons;
+
+public class IconsFrame extends Section {
+    @UI("//div[contains(@class, 'MuiGrid-grid-xs-8')][1]/*[name()='svg'][1]")
+    public Icons filled1;
+
+    @UI("//div[contains(@class, 'MuiGrid-grid-xs-8')][1]/*[name()='svg'][2]")
+    public Icons filled2;
+
+    @UI("//div[contains(@class, 'MuiGrid-grid-xs-8')][2]/*[name()='svg'][1]")
+    public Icons oulined1;
+
+    @UI("//*[name()='svg'][1]")
```

Comment:
this locator incorrect (many matches on page)
need to replace on .MuiSvgIcon-root